examples/example_app/example_app/forms.py

```diff
@@ -25,15 +25,15 @@ class ExampleModelFilterForm(BootstrapMixin, forms.Form):
 
     model = ExampleModel
     q = forms.CharField(required=False, label="Search")
-    name = forms.CharField(max_length=20, required=False)
+    name = forms.CharField(max_length=255, required=False)
```

Comment:
Can we add a constant somewhere (probably in `nautobot/core/constants.py`) and use it throughout instead of a bare `255`? This helps us to more explicitly assert this as our standard and also will make it easier if (heaven forbid) we ever need to change it in the future.